### PR TITLE
fix(userspace/libscap): use %lu for uint64 as modifier

### PR DESCRIPTION
**Pull request by loresuso**
**What type of PR is this?**

> Uncomment one (or more) `/kind <>` lines:

/kind bug

> /kind cleanup

> /kind design

> /kind documentation

> /kind failing-test

> /kind feature

**Any specific area of the project related to this PR?**

> Uncomment one (or more) `/area <>` lines:

> /area API-version

> /area build

> /area CI

> /area driver-kmod

> /area driver-bpf

> /area driver-modern-bpf

> /area libscap-engine-bpf

> /area libscap-engine-gvisor

> /area libscap-engine-kmod

> /area libscap-engine-modern-bpf

> /area libscap-engine-nodriver

> /area libscap-engine-noop

> /area libscap-engine-source-plugin

> /area libscap-engine-savefile

> /area libscap-engine-udig

/area libscap

> /area libpman

> /area libsinsp

> /area tests

> /area proposals

**Does this PR require a change in the driver versions?**

> /version driver-API-version-major

> /version driver-API-version-minor

> /version driver-API-version-patch

> /version driver-SCHEMA-version-major

> /version driver-SCHEMA-version-minor

> /version driver-SCHEMA-version-patch

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Review by jasondellaluce — APPROVED** [review]
/approve

/milestone 0.14.0

File: userspace/libscap/linux/scap_procs.c [hunk 1]
```diff
@@ -1386,7 +1386,7 @@ int32_t scap_linux_get_fdlist(struct scap_platform* platform, struct scap_thread
 	struct scap_linux_platform* linux_platform = (struct scap_linux_platform*)platform;
 
 	// We collect file descriptors only for the main thread
-	snprintf(proc_dir, sizeof(proc_dir), "%s/proc/%d/", scap_get_host_root(), tinfo->pid);
+	snprintf(proc_dir, sizeof(proc_dir), "%s/proc/%lu/", scap_get_host_root(), tinfo->pid);
```
Comment (Andreagit97) [inline userspace/libscap/linux/scap_procs.c:1389]: so in scap we use `uint64_t` but in sinsp, we use `int64_t` :confused: Wow

**Review by Andreagit97 — APPROVED** [review]
/approve

File: userspace/libscap/linux/scap_procs.c (hunk 1 above)
Comment (loresuso) [reply userspace/libscap/linux/scap_procs.c:1389]: I know ..

**Pull request merged**